src/main/java/spoon/reflect/visitor/CtInheritanceScanner.java

```diff
@@ -178,23 +195,23 @@ public void scanCtElement(CtElement e) {
 	 * Scans an abstract executable.
 	 */
 	public <R> void scanCtExecutable(CtExecutable<R> e) {
```

Comment:
CtExecutable inherits from CtNamedElement, CtGenericElement, CtTypedElement and CtTypeMember, why we don't need to scan them?